src/WebJobs.Extensions.DurableTask/EndToEndTraceHelper.cs

```diff
@@ -363,10 +403,18 @@ public static string LocalSlotName
                 functionType.ToString(),
                 ExtensionVersion,
                 IsReplay: isReplay);
+            if (this.ShouldReplayEventsBeLogged(false))
+            {
+                this.logger.LogInformation(
+                    "{instanceId}: Function '{functionName} ({functionType})' was resumed by a timer scheduled for '{expirationTime}'. IsReplay: {isReplay}. State: {state}. HubName: {hubName}. AppName: {appName}. SlotName: {slotName}. ExtensionVersion: {extensionVersion}. SequenceNumber: {sequenceNumber}.",
+                    instanceId, functionName, functionType, expirationTimeString, isReplay, FunctionState.TimerExpired,
+                    hubName, LocalAppName, LocalSlotName, ExtensionVersion, this.sequenceNumber++);
+            }
+        }
 
-            this.logger.LogInformation(
-                "{instanceId}: Function '{functionName} ({functionType})' was resumed by a timer scheduled for '{expirationTime}'. IsReplay: {isReplay}. State: {state}. HubName: {hubName}. AppName: {appName}. SlotName: {slotName}. ExtensionVersion: {extensionVersion}. SequenceNumber: {sequenceNumber}.",
-                instanceId, functionName, functionType, expirationTimeString, isReplay, FunctionState.TimerExpired, hubName, LocalAppName, LocalSlotName, ExtensionVersion, this.sequenceNumber++);
+        private bool ShouldReplayEventsBeLogged(bool isReplay)
```

Comment:
The method name doesn't quite represent what this method is used for. I think you should simply name it `ShouldLogEvent`.

Reply:
Great! Thank you very much! Shorter, simpler, better!